feature-libs/asm/customer-360/components/asm-customer-promotion-listing/asm-customer-promotion-listing.component.html

```diff
@@ -3,6 +3,61 @@ <h4 class="cx-asm-customer-promotion-listing-heading-text">
     {{ headerText }}
   </h4>
 </div>
+<ng-container *ngIf="isCustomerCoupon">
+  <div class="cx-asm-customer-promotion-listing-tabs">
+    <button
+      class="cx-tab-header"
+      [class.active]="activeTab === 0"
+      [textContent]="'customer360.customerCoupons.availableTab' | cxTranslate"
+      (click)="
+        refreshCustomerCouponList.emit({
+          assignable: true,
+          searchQuery: undefined
```

Comment:
no need pass searchQuery？

Reply:
here only change tab. no need to pass searchQuery